src/core_modules/capture-core/utils/validation/validators/form/isValidMinCharactersToSearch.ts

```diff
@@ -0,0 +1,30 @@
+const isValueBiggerThanMinCharactersToSearch = (value: string, minCharactersToSearch: number) => {
+    const trimmedValue = value.trim();
+    if (trimmedValue === '') {
+        return true;
+    }
+    return minCharactersToSearch <= trimmedValue.length;
+};
+
+export const isValidMinCharactersToSearch = (value: any, minCharactersToSearch: number) => {
+    if (value === undefined) {
+        return true;
+    }
+
+    if (typeof value == 'string') {
+        return isValueBiggerThanMinCharactersToSearch(value, minCharactersToSearch);
+    }
+
+    if ('from' in value && 'to' in value) {
+        return (
```

Comment:
Could the `from` and `to` properties reference an object as well? E.g., for a DATE_RANGE value type